stl_3.txt

```diff
@@ -9,3 +9,34 @@
 # Proszę dopisywać swoje pytania
 
 
+#Pytania Filip
+
+Iteracja oraz usuwanie elementu z wektora, co nalezy zrobic z iteratorem wektora po uzyciu funkcji erase:
+- Iteratora nie nalezy uzywac z metoda erase
+- Nie nalezy nic robic - Iterator automatycznie zostanie zaktualizowany aby wskazywal element n+1
++ metoda erase zwraca nowy iterator ktory nalezy zamienic ze starym 
+- zadne z powyzszych
+
+W jaki sposobro zorganizowana jest pamiec std::vector oraz std::list:
+- Sa tak samo zorganizowane jedyna roznica jest sposob zarzadzania elementami skladowymi
++ std::vector jest zorganizowany jako ciagly element pamieci, std::list jest rozganizowana jako "linked memory" kazdy element listy wskazuje na pamiec ktora moze byc dowolnie rozlozona. 
+- std::list jest zorganizowany jako ciagly element pamieci, std::vector jest rozganizowana jako "linked memory" kazdy element listy wskazuje na pamiec ktora moze byc dowolnie rozlozona.
+- zadne z powyzszych
+
+Co stanie sie gdy na std::vector zostanie wywolana metoda insert lub resize jednak aktualnie "przed nim" pamiec jest juz zarezerwowana:
```

Comment:
Nie rozumiem tego pytania, obawiam się że reszta też niekoniecznie musi je zrozumieć - co to jest pamięć przed.